source/style-guide/style/text-formatting.txt

```diff
@@ -1195,31 +1197,6 @@ elements. The following style differences are highlighted:
           - Client authentication is provided through a REST interface
             by using the ``GET`` method.
 
-   New terms
```

Comment:
[question]

Was this section deleted intentionally?

Reply:
The section is included, with a slight change in wording, under "Terms, new" starting around line 1551